src/mame/itech/itech32.cpp

```diff
@@ -1869,7 +1834,9 @@ void itech32_state::bloodstm(machine_config &config)
 
 void drivedge_state::drivedge(machine_config &config)
 {
-	/* basic machine hardware */
+	static constexpr XTAL TMS_CLOCK     = XTAL(40'000'000);          // TMS320C31 clocks on drivedge
+
+	// basic machine hardware
```

Comment:
Please don’t make this `static` – a local `constexpr` is better (avoids global data bloat).